db/helpers.go

```diff
@@ -122,6 +122,16 @@ func (ms *MongoStorage) initCollections(database string) error {
 	if ms.processBundles, err = getCollection("processBundles"); err != nil {
 		return err
 	}
+
+	// cspTokens collection
+	if ms.cspTokens, err = getCollection("cspTokens"); err != nil {
+		return err
+	}
+
+	// cspTokensStatus collection
+	if ms.cspTokensStatus, err = getCollection("cspTokensStatus"); err != nil {
+		return err
+	}
```

Comment:
Apart from initializing collections this is the file to create the corrsponing indexes.
Is there a cncrete reason why you did bring here the csp index:
https://github.com/vocdoni/saas-backend/blob/621acaf67f63e4366dee9e1e6413fdb712c7b472/csp/storage/mongodb.go#L139

Reply:
good catch! :raised_hands:

Reply:
i re-added the index in  73e2dd6dad28698b695b538321a7c19566c33aaa